.github/workflows/release.yml

```diff
@@ -0,0 +1,78 @@
+# This Workflow performs a full release of che-operator
+
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'version'
+        default: '7.x.y'
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Install operator courier
+        run: |
+          python -m pip install --upgrade pip
+          # pip install operator-courier
+
+      - name: Release operator
+        run: |
+          CHE_VERSION=${{ github.event.inputs.version }}
+          BRANCH=${CHE_VERSION%.*}.x
+
+          #preinstall operator-courier and operator-sdk
+          pip install operator-courier==2.1.7
+          pip install yq
+
+          set -e
+
+          OP_SDK_DIR=/opt/operator-sdk
+          mkdir -p $OP_SDK_DIR
+          wget https://github.com/operator-framework/operator-sdk/releases/download/v0.10.0/operator-sdk-v0.10.0-x86_64-linux-gnu -O $OP_SDK_DIR/operator-sdk 
+          chmod +x $OP_SDK_DIR/operator-sdk
+          export PATH="$PATH:$OP_SDK_DIR"
+
+          # get base32 util from che-release project
```

Comment:
do we still need this if we can just install the real base32?

Reply:
I'll use the same for consitency for now